audformat/core/utils.py

```diff
@@ -919,11 +937,21 @@ def join_schemes(
     if you want to combine databases
     with :meth:`audformat.Database.update`.
 
+    Joining schemes that use labels
+    from a misc table is not supported.
+    Please use
+    :meth:`audformat.Database.update`
+    instead.
+
     Args:
         dbs: sequence of databases
         scheme_id: scheme ID of a scheme with labels
             that should be joined
 
+    Raises:
+        ValueError: if joining schemes with different dtype
+            or only a subset of the schemes use labels from a misc table
```

Comment:
This is no longer true as we don't support misc tables, so maybe:

```suggestion
        ValueError: if scheme labels are of different dtype
            or not ``list`` or ``dict``
```